tests/test_pe_source.py

```diff
@@ -0,0 +1,104 @@
+"""Tests for the pe-source module."""
+
+# Standard Python Libraries
+import logging
+import sys
+from unittest.mock import patch
+
+# Third-Party Libraries
+import pytest
+
+# cisagov Libraries
+import pe_source.pe_scripts
+
+log_levels = (
+    "debug",
+    "info",
+    "warning",
+    "error",
+    "critical",
+)
+
+
+PROJECT_VERSION = pe_source.__version__
+
+
+# TODO: Replace current dummy test with useful tests
+# Issue - https://github.com/cisagov/pe-reports/issues/3#issue-909531010
+
+
+def test_reports_stdout_version(capsys):
+    """Verify that version string sent to stdout agrees with the module version."""
+    with pytest.raises(SystemExit):
+        with patch.object(sys, "argv", ["bogus", "--version"]):
+            pe_source.pe_scripts.main()
+    captured = capsys.readouterr()
+    assert (
+        captured.out == f"{PROJECT_VERSION}\n"
+    ), "standard output by '--version' should agree with module.__version__"
+
+
+def test_reports_running_as_module(capsys):
```

Comment:
Modify test name to match module being tested:
```suggestion
def test_source_running_as_module(capsys):
```